sql/mysql/full.sql

```diff
@@ -78,6 +78,7 @@ CREATE TABLE `characters` (
   `x` float NOT NULL,
   `y` float NOT NULL,
   `z` float NOT NULL,
+  `o` float NOT NULL,

```

Comment:
Wrong patch?

Reply:
Well, kind of, I didn't really know what to do with the change so I just committed it as it's own commit, I can get rid of it and place it in it's own PR.

Reply:
Would be ideal.

Reply:
Why isn't o called facing here? I know it's for orientation, but using the internal names is a plus when reversing.

Reply:
because facing isn't the internal name as far as I can tell, neither is x, y or z; 
![image](https://user-images.githubusercontent.com/1907820/59564084-94312200-9042-11e9-9f78-0e4153d3ed19.png)

Reply:
(I am guessing POSSTAT0 is map, 1, 2, and 3 is x, y and z and 4 is orientation)

Reply:
Unless you are referring to the internal GO names, in which case x, y and z should be renamed pos_x, pos_y and pos_z as well;
![image](https://user-images.githubusercontent.com/1907820/59564148-ab244400-9043-11e9-8917-6291bb19f7b4.png)